### PR TITLE
COMMON: Fix code style in makefile.

### PR DESCRIPTION
**Pull request by fedor4ever**
That strange makefile structure break Symbian makefile parser.

### CODE REVIEW

File: common/module.mk [hunk 1]
```diff
@@ -52,7 +52,8 @@ MODULE_OBJS := \
 	winexe_ne.o \
 	winexe_pe.o \
 	xmlparser.o \
-	zlib.o
+	zlib.o \
+	text-to-speech.o
```
Comment (criezy) [inline common/module.mk:56]: This list is mostly alphabetical, so it would probably make more sense to move it just after `system.o`.

**Comment by criezy** [comment]
The current structure does not seem particularly strange to me, but I see no issue with moving the text-to-speech.o to the main list. It is currently separate for historical reason because it used to be added only when `USE_TTS` is defined, but this was changed recently (in 9d82fa5).

However as noted in my inline comment I think it would be better to keep the list alphabetical and move it accordingly (and not at the end of the list).

**Comment by fedor4ever** [comment]
@criezy , done!

**Comment by sev-** [comment]
Thanks, merging.

**Pull request merged**